src/lib/promiseHelper.js

```diff
@@ -4,20 +4,17 @@ export const filterPromises = (arr, filter) =>
    Promise.all(arr.map(entry => filter(entry)))
      .then(bits => arr.filter(entry => bits.shift()));
 
-export const resolvePromiseProperties = obj =>
-  (new Promise((resolve, reject) => {
-    // Get the keys which represent promises
-    const promiseKeys = Object.keys(obj).filter(
-      key => obj[key] instanceof Promise);
+export const resolvePromiseProperties = (obj) => {
+  // Get the keys which represent promises
+  const promiseKeys = Object.keys(obj).filter(
+    key => obj[key] instanceof Promise);
```

Comment:
Could've sworn we swapped this in the original PR, but we should simply check for a `then` method to determine if it's a promise.